source/Octopus.Tentacle.CommonTestUtils/TemporaryDirectory.cs

```diff
@@ -27,7 +28,7 @@ string GetTempBasePath()
             var path = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData, Environment.SpecialFolderOption.DoNotVerify);
             fileSystem.EnsureDirectoryExists(path);
 
-            path = Path.Combine(path, Assembly.GetEntryAssembly() != null ? Assembly.GetEntryAssembly()!.GetName().Name! : "Octopus");
+            path = Path.Combine(path, Assembly.GetEntryAssembly()?.GetName()?.Name ?? "Octopus");
```

Comment:
were you getting null ref's before?

Reply:
no, but I think this is a more modern way to handle this statement

Reply:
So the change of behaviour is intended here? Now it will use `Octopus` if anything in the chain is `null`

Reply:
Let me check the nullability annotations, but I think this is a safe change

Worth mentioning that this is just for creating a temp directory, but let me double check

Reply:
I double checked and I probably just have some extra `?` in the chain I don't actually need. The nullability check on the `GetEntryAssembly()` is required, but from then on, `GetName()` will return a non-null type and then `Name` is nullable. 

There is a slight behavioural difference in this case as previously, if you had an entry assembly, but `.Name` returned null, it would blow up due to the `!`.

The change now will handle this case.